### ISSUE TITLE
Portrait orientation not supported in Admin Screen

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#847]
**Describe the bug**
Portrait orientation not supported in Admin Screen

**To Reproduce**

Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test2&path=%2FSites%2Ftest2%2F&pathType=page)
Press F12 key to open Chrome DevTools popup.
On the Devtools popup, enable the browser emulator by clicking the "Toggle device toolbar" icon in the top left.
Th emulator screen will be opened.
On the emulator screen, click on the rotate button to switch between Landscape and Portrait orientation to see the issue.

**Expected behavior**
Portrait orientation should be supported.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by rgarg24** [comment]
https://www.w3.org/TR/WCAG21/#orientation (Accesibility)

**Comment by natechadwick-intsof** [comment]
Issue migrated to: https://github.com/intersoftdatalabs-in/percussioncms/issues/945

**Comment by natechadwick-intsof** [comment]
Issue closed and migrated to: https://github.com/intersoftdatalabs-in/percussioncms/issues/945
